src/schema/v2/homeView/index.ts

```diff
@@ -11,12 +11,12 @@ import {
   connectionWithCursorInfo,
   paginationResolver,
 } from "../fields/pagination"
-import { HomeViewSectionType } from "./HomeViewSection"
+import { HomeViewGenericSectionInterface } from "./HomeViewSection"
 import { getSectionsForUser } from "./getSectionsForUser"
 import { registry } from "./sections"
 
 const SectionsConnectionType = connectionWithCursorInfo({
-  nodeType: HomeViewSectionType,
+  nodeType: HomeViewGenericSectionInterface,
```

Comment:
Shouldn't this be the union?

Reply:
No, this is an intentional change from Union > Interface.